src/inmanta/app.py

```diff
@@ -90,18 +90,15 @@ def __init__(
         no_color: bool = False,
         force_color: bool = False,
         defaults: typing.Optional[typing.Mapping[str, typing.Any]] = None,
-        timed: bool = False,
     ):
         super().__init__(
             fmt, datefmt, style, log_colors, reset, secondary_log_colors, validate, stream, no_color, force_color, defaults
         )
-        self.timed = timed
+        self.fmt = fmt
 
     def get_header_length(self, record):
         """Get the header length of a given record."""
-        log_format = "%(asctime)s " if self.timed else ""
-        log_format += "%(name)-25s%(levelname)-8s%(message)s"
-        formatter = logging.Formatter(fmt=log_format)
+        formatter = colorlog.ColoredFormatter(fmt=self.fmt, reset=False, force_color=False, no_color=True)
```

Comment:
I think we should additionally pass all other parameters as is.

Reply:
indeed good catch :)

Reply:
No if we used the color codes here the length is not right:
"\x1B[30mHello" would return a length of more then 5 but the displayed string has only length 5 and we only care about that part as the color codes are not displayed.

Reply:
I don't mean these parameters. For these I agree we should hard set them, that's the whole purpose here (perhaps add a comment about this?). But for all other parameters that we don't care about, shouldn't we make sure to set them to be the same?

Reply:
Ho I see! :) I'll add them, it can do no harm :)